diff --git b/lib/rubygems/vendor/molinillo/lib/molinillo/dependency_graph.rb a/lib/rubygems/vendor/molinillo/lib/molinillo/dependency_graph.rb
index 16712899b..731a9e3e9 100644
--- b/lib/rubygems/vendor/molinillo/lib/molinillo/dependency_graph.rb
+++ a/lib/rubygems/vendor/molinillo/lib/molinillo/dependency_graph.rb
@@ -1,6 +1,6 @@
 # frozen_string_literal: true
 
-require_relative '../../../tsort/lib/tsort'
+require_relative '../../../../vendored_tsort'
 
 require_relative 'dependency_graph/log'
 require_relative 'dependency_graph/vertex'
